### ISSUE TITLE
mumble-dark theme icons color

### ISSUE THREAD

**Issue by stderr-to-devnull** [jagannatharjun/qbt-theme#29]
After upgrading to qbittorrent 4.3.0 and downloading the new mumble-dark theme, I noticed that almost all icons are now pink. Is this by design? Can the color be modified? Can the actual look of the new theme be reflected in an updated screenshot?

**Comment by mzso** [comment]
I think its some bug. I have orange for inactive upload/downlad and paused as well.
Looks rather bad:
![kép](https://user-images.githubusercontent.com/4479090/96425734-a30e8280-11fc-11eb-85fa-4b96bd90c809.png)

**Comment by Hvalfisk** [comment]
The orange stalled up and down arrows are really hard to tell apart.

**Comment by mzso** [comment]
Is this issue because of new QBT or the new theme? I upgraded simultaneously.

**Comment by jagannatharjun** [comment]
> Can the actual look of the new theme be reflected in an updated screenshot?

what're you saying it's already updated.

> Is this issue because of new QBT or the new theme? I upgraded simultaneously.

What's the issue?

**Comment by jagannatharjun** [comment]
> I  noticed that almost all icons are now pink.  Is this by design? Can the color be modified?

yes starting with qbit 4.3.0 icons can be changed from theme files.

**Comment by stderr-to-devnull** [comment]
OK so from what I deduced, icon colors were a design choice. The screenshot shows some of the updated color for icons but not all elements (done, uploading, sidebar with trackers list etc) so this is why it appears to look different (the UI in the screenshot is a lot cleaner, meaning one torrent with one status). I would say it no longer looks that good when having all elements in the UI + full list of torrents but this is subjective of course.

> yes starting with qbit 4.3.0 icons can be changed from theme files.
Can you help with showing us how to do this?

**Comment by mzso** [comment]
> 
> > Is this issue because of new QBT or the new theme? I upgraded simultaneously.
> 
> What's the issue?

Most category/state icons being orange. (See screenshot) In my case anyway. Someone else said it's pink.

**Comment by jagannatharjun** [comment]
> Can you help with showing us how to do this?

1. Download files from here https://github.com/jagannatharjun/qbt-theme
2. Download all files from here https://github.com/nowshed-imran/qBittorrent/tree/redo/src/icons and put it in Builds\icons\nowshed
3. run this https://github.com/jagannatharjun/qbt-theme/blob/master/Builds/build-mumble-dark.bat

this line changes colors https://github.com/jagannatharjun/qbt-theme/blob/master/Builds/build-mumble-dark.bat#L9

**Comment by TheArgentinian** [comment]
I really dislike the new icons. Too distracting.

**Comment by mzso** [comment]
@TheArgentinian 

> I really dislike the new icons. Too distracting.

You also get orange icons?
What's annoying is that several different ones are the same color. Which makes no sense design wise, not practical at all...

**Comment by jagannatharjun** [comment]
@TheArgentinian @mzso 
try these 
[new-icons.zip](https://github.com/jagannatharjun/qbt-theme/files/5417781/new-icons.zip)

**Comment by TheArgentinian** [comment]
> @TheArgentinian
> 
> > I really dislike the new icons. Too distracting.
> 
> You also get orange icons?

Yes



> @TheArgentinian @mzso
> try these
> [new-icons.zip](https://github.com/jagannatharjun/qbt-theme/files/5417781/new-icons.zip)

They are still all colored like a rainbow. They should blend with the background, not pop-up, do you know what I mean?

**Comment by ArtexJay** [comment]
I'm not a fan of the pinkish default color icon. I'd rather you have used the blue ones you did for breeze-dark/darkstylesheet. Those are better than the pink ones.

**Comment by mzso** [comment]
> > @TheArgentinian @mzso
> > try these
> > [new-icons.zip](https://github.com/jagannatharjun/qbt-theme/files/5417781/new-icons.zip)
> 
> They are still all colored like a rainbow. They should blend with the background, not pop-up, do you know what I mean?

I don't think so. All differing ones should have a different color. Monochrome/grey nonsense is the worst.

**Comment by japanesephundroid** [comment]
> > > @TheArgentinian @mzso
> > > try these
> > > [new-icons.zip](https://github.com/jagannatharjun/qbt-theme/files/5417781/new-icons.zip)
> > 
> > 
> > They are still all colored like a rainbow. They should blend with the background, not pop-up, do you know what I mean?
> 
> I don't think so. All differing ones should have a different color. Monochrome/grey nonsense is the worst.

I agree with this, color icons are easier to differentiate, with colors that can be easily associated with the action.

I liked **mumble-dark-nowshed.qbtheme** in the **new-icons.zip**.

**Comment by xiao-mantou** [comment]
Rolling back to V1, everything is fine.

**Comment by mzso** [comment]
[**@jagannatharjun**](https://github.com/jagannatharjun) commented on [2020. okt. 21. 19:41 CEST](https://github.com/jagannatharjun/qbt-theme/issues/29#issuecomment-713741919 "2020-10-21T17:41:09Z - Replied by Github Reply Comments"):
> [@TheArgentinian](https://github.com/TheArgentinian) [@mzso](https://github.com/mzso)  
> try these  
> [new-icons.zip](https://github.com/jagannatharjun/qbt-theme/files/5417781/new-icons.zip)

Thanks!
Much better this way. To be clear, were the colors set to orange for several different icons on purpose, by hand? (or was it some glitch?)
By the way, the difference between recolored and plain "nowshed" (BTW what is "nowshed"? :) ) is that some of the blue colors are replaced with purple? I say some, because scrollbar, stopped torrent color, selected tab is still blue. I actually prefer blue, I find that pale purple a little weird. (To be honest I'd prefer warm colors for dark themes. More natural.)

One idea: Stuck download/upload colors could be a dimmer/less-saturated/darker shade of the normal upload/download colors. So people could associate one color with upload/download, but also immediately tell whether it's active or not.

**Comment by jagannatharjun** [comment]
> (BTW what is "nowshed"? :) )

creator of icons, these icons are proposed for the next default icons for qbittorrent, "recolored" icons are recolored ones. 

Do give your opinions there too https://github.com/qbittorrent/qBittorrent/pull/12965

**Comment by ArtexJay** [comment]
> @TheArgentinian @mzso
> try these
> [new-icons.zip](https://github.com/jagannatharjun/qbt-theme/files/5417781/new-icons.zip)

Can you update these files with the fixed progress bar? The newest qbittorrent update messes up the progress bar a bit, it's not centered anymore. as mentioned in #35

**Comment by RabidWolf** [comment]
@jagannatharjun

Will the mumble themes, ones with nowshed icons, be updated soon?

**Comment by mzso** [comment]
[**@RabidWolf**](https://github.com/RabidWolf) commented on [2020. dec. 2. 21:37 CET](https://github.com/jagannatharjun/qbt-theme/issues/29#issuecomment-737481421 "2020-12-02T20:37:46Z - Replied by Github Reply Comments"):
> [@jagannatharjun](https://github.com/jagannatharjun)
> 
> Will the mumble themes, ones with nowshed icons, be updated soon?

For heaven's sake. The main version still has identical icon colors.

**Comment by ArtexJay** [comment]
@jagannatharjun the superior version for Mumble dark theme that you posted in the zips here, haven't been updated to version 2.1 yet.

**Comment by mzso** [comment]
[**@ArtexJay**](https://github.com/ArtexJay) commented on [2020. dec. 12. 08:49 CET](https://github.com/jagannatharjun/qbt-theme/issues/29#issuecomment-743719738 "2020-12-12T07:49:27Z - Replied by Github Reply Comments"):
> [@jagannatharjun](https://github.com/jagannatharjun) the superior version for Mumble dark theme that you posted in the zips here, haven't been updated to version 2.1 yet.

Yeah. Updating that would be nice. Or just adding the icon set to Mumble dark proper.

**Comment by mzso** [comment]
[**@jagannatharjun**](https://github.com/jagannatharjun) commented on [2020. okt. 21. 07:04 CEST](https://github.com/jagannatharjun/qbt-theme/issues/29#issuecomment-713305420 "2020-10-21T05:04:48Z - Replied by Github Reply Comments"):
> > Can you help with showing us how to do this?
> 
> 1.  Download files from here [https://github.com/jagannatharjun/qbt-theme](https://github.com/jagannatharjun/qbt-theme)
> 2.  Download all files from here [https://github.com/nowshed-imran/qBittorrent/tree/redo/src/icons](https://github.com/nowshed-imran/qBittorrent/tree/redo/src/icons) and put it in Builds\\icons\\nowshed
> 3.  run this [https://github.com/jagannatharjun/qbt-theme/blob/master/Builds/build-mumble-dark.bat](https://github.com/jagannatharjun/qbt-theme/blob/master/Builds/build-mumble-dark.bat)
> 
> this line changes colors [https://github.com/jagannatharjun/qbt-theme/blob/master/Builds/build-mumble-dark.bat#L9](https://github.com/jagannatharjun/qbt-theme/blob/master/Builds/build-mumble-dark.bat#L9) Currently it only changes the primary color to pink, you can play with it and you need python installed

So this doesn't work at all. I guess it has a bunch of dependencies, that I don't have. (Also it seems finicky about paths. Since it gave different errors at different locations.)

**Comment by mzso** [comment]
@jagannatharjun 
So, did development on these dark themes effectively cease? The last version with identical colors for most icons I find poorly usable. And the one with the updated icons shared here still have visual glitches on the progress bar.

**Comment by jagannatharjun** [comment]
> did development on these dark themes effectively cease? 

hmm, maybe yes, I personally liked the current themes and I neither have the time nor any kind of motivation to change it. progress bar text should be fixed in the current master. though you have to compile the files yourself.

**Comment by mzso** [comment]
[**@jagannatharjun**](https://github.com/jagannatharjun) commented on [2021. márc. 13. 18:33 CET](https://github.com/jagannatharjun/qbt-theme/issues/29#issuecomment-798674126 "2021-03-13T17:33:56Z - Replied by Github Reply Comments"):
> > did development on these dark themes effectively cease?
> 
> hmm, maybe yes, I personally liked the current themes and I neither have the time nor any kind of motivation to change it. progress bar text should be fixed in the current master. though you have to compile the files yourself.

I don't know how to compile it though.
Though the PB is fixed in the previous version as well, but that doesn't have the updated icons. Are you implying that since then the icons were updated as well?

**Comment by jagannatharjun** [comment]
check https://github.com/jagannatharjun/qbt-theme/blob/master/mumble-dark-nowshed.qbtheme for no pink icons

**Comment by mzso** [comment]
[**@jagannatharjun**](https://github.com/jagannatharjun) commented on [2021. szept. 18. 11:46 CEST](https://github.com/jagannatharjun/qbt-theme/issues/29#issuecomment-922250256 "2021-09-18T09:46:30Z - Replied by Github Reply Comments"):
> check [https://github.com/jagannatharjun/qbt-theme/blob/master/mumble-dark-nowshed.qbtheme](https://github.com/jagannatharjun/qbt-theme/blob/master/mumble-dark-nowshed.qbtheme) for no pink icons

Thanks, though it doesn't help with my problem. All this orange instead of the separately colored categories of the older (and now glitched) variant you shared before is not very practical.

**Comment by jagannatharjun** [comment]
@mzso you can give your opinion https://github.com/qbittorrent/qBittorrent/pull/15453, the icon set will be default for qbittorrent 4.4.0+ release.
